.travis.yml

```diff
@@ -79,6 +79,7 @@ script:
   - make -j$JOBS > build.log 2>&1 || (cat build.log && exit 1)
   - sudo make PATH=$PATH install_sw > build.log 2>&1 || (cat build.log && exit 1)
   - cd ../mockeagain/ && make CC=$CC -j$JOBS && cd ..
+  - pushd ../lua-nginx-module && curl https://patch-diff.githubusercontent.com/raw/openresty/lua-nginx-module/pull/1145.patch | git apply && popd
```

Comment:
You can just change the `git clone` line to clone your branch instead. It's easier.

Reply:
thx.